doc/manual/manual.tex

```diff
@@ -306,14 +306,14 @@ \subsection{Referencing \aspect{}}
 \end{lstlisting}
 Please also cite the User's Manual:
 \begin{lstlisting}[frame=single,language=tex]
-@Manual{aspectmanual,
-  title =        {\textsc{ASPECT}: {Advanced Solver for Problems in Earth's
-                  ConvecTion}},
-  author =       {W. Bangerth and J. Dannberg and
-                  R. Gassm{\"o}ller and T. Heister and others},
-  organization = {Computational Infrastructure for Geodynamics},
-  year =         {2017},
-  url =          {https://aspect.dealii.org/}
+@Article{aspectmanual,
+  Title         = {{\textsc{ASPECT}: Advanced Solver for Problems in Earth's ConvecTion User Manual}},
+  Author        = {W. Bangerth and J. Dannberg and R. Gassm{\"o}ller and T. Heister and others},
+  Year          = {2017},
+  Month         = {4},
+  Doi           = {10.6084/m9.figshare.4865333},
+  Note          = {doi:10.6084/m9.figshare.4865333},
+  Url           = {https://doi.org/10.6084/m9.figshare.4865333}
```

Comment:
I think both DOI and URL are usually capitalized. bibtex doesn't care either way, so let's go with the standard.